clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/MoveExecutor.scala

```diff
@@ -100,20 +100,27 @@ class MoveExecutor[TI <: TransferIndex](moveCommand: MoveCommand[TI],
     *
     * We flatten out the metadata instances for pre- and post-move into Maps
     * from string keys to Any values, then flatMap away Options by extracting
-    * the underlying values from Somes and removing Nones.
+    * the underlying values from Somes and removing Nones, then finally filter
+    * away any non-URI values.
     *
     * The Option-removal is needed so we can successfully pattern-match on URI
     * later when building up the list of (preMove -> postMove) paths. Without it,
     * matching on Option[URI] fails because of type erasure.
     */
-  private def flattenMetadata(
+  private def extractPaths(
```

Comment:
Now that this method has a better name, maybe the comment can be updated too. The comment now describes how the method works, which is helpful, but it should start with a description of what it does. E.g. info for callers goes first, info for method maintainers goes second (or goes in a different, non-scaladoc comment).